samples/subsys/display/lvgl/src/main.c

```diff
@@ -36,6 +36,13 @@ static void button_isr_callback(const struct device *port,
 }
 #endif
 
+static void button_click_callback(lv_event_t *e)
```

Comment:
A last nite, sorry! Perhaps name it as follows, otherwise the naming of the two callback is unnecessarily confusing: 

```suggestion
static void lv_btn_click_callback(lv_event_t *e)
```

Reply:
I am always thankful for feedback/comments.

Reply:
fixed